resources/lib/sony.py

```diff
@@ -372,13 +372,13 @@ def put_resume_time(self, airing_id, channel_id, program_id, series_id, tms_id):
         payload += '"tms_id":"'+tms_id+'",'
         payload += '"airing_id":'+airing_id+','
         payload += '"last_watch_date":"2017-11-13T06:54:419Z",'

```

Comment:
This date will have to be set to the current date. Most likely utc now converted to the format displayed.